packages/nimble-components/src/table-column/number-text/models/volt-scale-formatter.ts

```diff
@@ -0,0 +1,25 @@
+import { ManuallyTranslatedUnitScaleFormatter } from './manually-translated-unit-scale-formatter';
+import { UnitPrefix, metricPrefixes } from './unit-prefix';
+import { UnitTranslation } from './unit-translation';
+
+/**
+ * Formatter for numbers with voltage units
+ */
+export class VoltScaleFormatter extends ManuallyTranslatedUnitScaleFormatter {
```

Comment:
I'd vote for this being VoltUnitScaleFormatter, it being ManuallyTranslatted is an implementation detail, not what this is implementing (a UnitScaleFormatter)